lib/pure/net.nim

```diff
@@ -535,7 +534,10 @@ when defineSsl:
         raiseSSLError("Verification of private key file failed.")
 
   proc newContext*(protVersion = protSSLv23, verifyMode = CVerifyPeer,
-                   certFile = "", keyFile = "", cipherList = "ALL",
+                   certFile = "", keyFile = "",
+                   # XXX: update the OpenSSL that we ship with Windows for the
+                   #      next release.
+                   cipherList = when not defined(windows): CiphersIntermediate else: CiphersOld,
```

Comment:
This is added as a temporary fix for Windows. The OpenSSL DLLs for Windows should be updated to at least 1.0.2 (which is the oldest version that can be consider to be "secure" enough, though it's also reached EOL).